test/protractor.conf.js

```diff
@@ -89,7 +89,9 @@ exports.config = {
   // run.
   suites: {
     // smoke: 'spec/smoketests/*.js',
-    rest_api: 'integration/rest_api/*.js' // This suite of tests should only require a running master api, it should not require a node
+    'add-to-project': 'integration/features/user_adds_template_to_project.spec.js',
+    'create-project': 'integration/features/user_creates_project.spec.js' // This suite of tests should only require a running master api, it should not require a node
```

Comment:
these feel a bit specific for suite names to me, i feel like the split is typically going to be tests that require a working node and tests that don't.  I can't remember the format for what a suite name can point to, is it only a single regexp string, or can it also be an array of strings?